image/docker_schema1.go

```diff
@@ -112,6 +113,17 @@ func (m *manifestSchema1) ConfigBlob() ([]byte, error) {
 	return nil, nil
 }
 
+// Config returns the image configuration as per OCI v1 image-spec
+func (m *manifestSchema1) Config() (*imgspecv1.Image, error) {
+	// Best effort to have an OCI configuration from a docker v2s1 manifest.
+	// we don't compute diffIDs or other v2s2 things here.
```

Comment:
I do agree that blob size and DiffID computation in here would be _very_ awkward, and is on balance probably best left out, OTOH it is a _really_ surprising aspect of the data output and needs to be _very prominently_ featured in the interface documentation in `types.go` (if not in the actual method name… that would be overkill I guess).